web/src/main/java/com/bazaarvoice/emodb/web/auth/DefaultRoles.java

```diff
@@ -39,6 +39,15 @@
             Permissions.createSorTable(NON_SYSTEM_TABLE),
             Permissions.setSorTableAttributes(NON_SYSTEM_TABLE)),
 
+    // sor_standard with excluding pii tables/placements.
+    // Pii tables contain publicly identifiable information, so in a way they are special compared to the rest of the sor tables.
```

Comment:
Nitpick: _personally_ identifiable information; the nature of PII is that it _isn't_ public.